### ISSUE TITLE
Create and add an Accessibility Statement

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/web-map-doc#34]
Copy & paste from the issue I raised in https://github.com/Leaflet/Leaflet/issues/7537:

> Inaccessibility may be a legal liability under certain [international laws and policies](https://www.w3.org/WAI/policies/). End-users of Leaflet maps have the right to an accessible experience, and implementers (e.g. organizations, individual developers) need to know if they meet those requirements. Documentation should be provided, describing if - and to what extent - Leaflet is accessible.
>
> The <span title="World Wide Web Consortium">W3C</span> Web Accessibility Initiative (WAI) provides a [tool to generate](https://www.w3.org/WAI/planning/statements/generator/#create) an [accessibility statement](https://www.w3.org/WAI/planning/statements/) that could be published to the Leaflet website (reachable through the Leaflet site link in the attribution).

We should probably have an accessibility statement. I have generated a draft accessibility statement to give you an idea:

<details><summary>Draft Accessibility Statement</summary><br>

> <h2>Accessibility Statement for <span class="basic-information website-name">Web Map Custom Element</span></h2>
> <p>
> 	This is an accessibility statement from <span class="basic-information organization-name">Maps for HTML Community Group</span>.
> </p>
> <h3>Measures to support accessibility</h3>
> <p>
> 	<span class="basic-information organization-name">Maps for HTML Community Group</span>
> 	takes the following measures to ensure accessibility of
> 	<span class="basic-information website-name">Web Map Custom Element</span>:
> </p>
> <ul class="organizational-effort accessibility-measures">
> 	<li>Include accessibility as part of our mission statement.</li>
> 	<li>Appoint an accessibility officer and/or ombudsperson.</li>
> 	<li>Provide continual accessibility training for our staff.</li>
> 	<li>Assign clear accessibility goals and responsibilities.</li>
> 	<li>Employ formal accessibility quality assurance methods.</li>
> </ul>
> <h3>Conformance status</h3>
> <p>
> 	The <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines (WCAG)</a> defines requirements for designers and developers to improve accessibility for people with disabilities. It defines three levels of conformance: Level A, Level AA, and Level AAA.
> 	<span class="basic-information website-name">Web Map Custom Element</span>
> 	is
> 	<span class="basic-information conformance-status" data-printfilter="lowercase">partially conformant</span>
> 	with
> 	<span class="basic-information conformance-standard"><span data-negate="">WCAG 2.1 level AA</span>.</span>
> 	<span>
> 	<span class="basic-information conformance-status">Partially conformant</span>
> 	means that
> 	<span class="basic-information conformance-meaning">some parts of the content do not fully conform to the accessibility standard</span>.
> </span>
> </p>
> <h3>Feedback</h3>
> <p>
> 	We welcome your feedback on the accessibility of
> 	<span class="basic-information website-name">Web Map Custom Element</span>.
> 	Please let us know if you encounter accessibility barriers on
> 	<span class="basic-information website-name">Web Map Custom Element</span>:
> </p>
> <ul class="basic-information feedback h-card">
> 	<li>
> 	E-mail:
> 	<a class="email u-email" href="mailto:public-maps4html@w3.org">public-maps4html@w3.org</a>
> </li>
> 	<li class="contact-other p-note">Github repository: https://github.com/Maps4HTML/Web-Map-Custom-Element/issues | Twitter: @maps4html</li>
> </ul>
> <h3>Compatibility with browsers and assistive technology</h3>
> <p>
> 	<span class="basic-information website-name">Web Map Custom Element</span>
> 	is designed to be compatible with the following assistive technologies:
> </p>
> <ul class="technical-information compatible-environments">
> 	<li>Screen readers (partial compatibility)</li>
> </ul>
> <p>
> 	<span class="basic-information website-name">Web Map Custom Element</span>
> 	is not compatible with:
> </p>
> <ul class="technical-information incompatible-environments">
> 	<li>Internet Explorer, and other browsers that do not support Autonomous Custom Elements: https://caniuse.com/custom-elementsv1</li>
> </ul>
> <h3>Technical specifications</h3>
> <p>
> 	Accessibility of
> 	<span class="basic-information website-name">Web Map Custom Element</span>
> 	relies on the following technologies to work with the particular combination of web browser and any assistive technologies or plugins installed on your computer:
> </p>
> <ul class="technical-information technologies-used">
> 	<li>HTML</li>
> 	<li>WAI-ARIA</li>
> 	<li>CSS</li>
> 	<li>JavaScript</li>
> 	<li>MapML</li>
> </ul>
> <p>These technologies are relied upon for conformance with the accessibility standards used.</p>
> <h3>Limitations and alternatives</h3>
> <p>
> 	Despite our best efforts to ensure accessibility of
> 	<span class="basic-information website-name">Web Map Custom Element</span> , there may be some limitations. Below is a description of known limitations, and potential solutions. Please contact us if you observe an issue not listed below.
> </p>
> <p>
> 	Known limitations for
> 	<span class="basic-information website-name">Web Map Custom Element</span>:
> </p>
> <ol class="technical-information accessibility-limitations">
> 	<li><strong>Panning and zooming of the map presentation</strong>: No feedback is provided to assistive technologies on pan and zoom because of reliance on underlying technologies that has done little or no prior work on this area. Issue discussion: https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/396. .</li>
> </ol>
> <h3>Assessment approach</h3>
> <p>
> 	<span class="basic-information organization-name">Maps for HTML Community Group</span>
> 	assessed the accessibility of
> 	<span class="basic-information website-name">Web Map Custom Element</span>
> 	by the following approaches:
> </p>
> <ul class="technical-information assessment-approaches">
> 	<li>Self-evaluation</li>
> </ul>
> <h3>Other evidence</h3>
> <p>Other related evidence for <span class="basic-information website-name">Web Map Custom Element</span> is available at:</p>
> <ul class="technical-information related-evidence-other">
> 	<li>https://github.com/Maps4HTML/Web-Map-Custom-Element/issues?q=is%3Aissue+label%3Aaccessibility</li>
> </ul>
> <hr noshade="noshade">
> <h3>Date</h3>
> <p>
> 	This statement was created on
> 	<span class="basic-information statement-created-date">22 July 2021</span>
> 	using the <a href="https://www.w3.org/WAI/planning/statements/">W3C Accessibility Statement Generator Tool</a>.
> </p>

</details>

There are certainly additions and general improvements to be made, but what are your thoughts on adding such an accessibility statement. I suppose there are legalities involved with publishing such a statement, so it'd have to be thoroughly reviewed once a final version has been made.

The idea is to create a web page on the site to hold the accessibility statement, linked from the site's footer section.

**Comment by Malvoz** [comment]
The draft a11y statement in the comment above closely resembles the [complete example](https://www.w3.org/WAI/planning/statements/complete-example/), should tone it down closer to the [minimal example](https://www.w3.org/WAI/planning/statements/minimal-example/).
